### PR TITLE
[DOCS-7435] Add external Hazelcast clustering for ACS 23.1

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

File: content-services/latest/admin/cluster.md [hunk 1]
```diff
@@ -332,9 +333,58 @@ This topic describes the instructions for installing and configuring Solr nodes
...
+### Set up repository clustering via external Hazelcast
+
+Managing your own Hazelcast cluster allows you to scale repository and Hazelcast instances independently, while also allowing you to manage resources allocation on an individual level. This kind of setup provides more fine-grained control at the cost of more necessary preliminary work.
+
+1. Prepare a valid Hazelcast XML configuration file including all caches definitions required by the repository. This can be mostly auto-generated for you by doing the following:
+
+    1. Locate the `caches.properties` file within the `alfresco.war:WEB-INF\lib\alfresco-repository-*.jar` archive.
+    2. Copy the `caches.properties` file to an easily accessible directory.
+    3. Clone the [alfresco-enterprise-repo.git](https://github.com/Alfresco/alfresco-enterprise-repo.git){:target="_blank"} repository locally.
+    4. Move into the previously cloned repository's directory.
```
Comment (dsibilio) [inline content-services/latest/admin/cluster.md:352]: I was thinking about this particular point for a bit and I think there's been a misunderstanding related to how we can hand over this file to our customers. I believe even our enterprise customers don't have SSO access to the Alfresco GitHub organization so they wouldn't be able to follow these steps.

Some alternatives could be:
- embedding the whole file in the docs: https://raw.githubusercontent.com/Alfresco/alfresco-enterprise-repo/master/repository/scripts/hazelcast-init/generate-hazelcast-config.py?token=GHSAT0AAAAAACG2ONI63UEFRHCBY73F4JMEZKUSEXQ _(~140 lines of code as of today)_
- moving this file to a public repository such as e.g.: `alfresco-community-repo` instead _(this also needs adjusting the CI/CD of enterprise-repo to leverage this file coming from community-repo)_

@anxumalo @HeinRagas do you have any preference between the two options if both prove to be applicable?

**Review by dsibilio — CHANGES_REQUESTED** [review]

File: content-services/latest/admin/cluster.md (hunk 1 above)
Comment (anxumalo) [reply content-services/latest/admin/cluster.md:352]: @dsibilio I overlooked that this is a private repo issue so good catch! If we add the whole file to the docs, there's a risk of it becoming out of sync, or mis-formatted from copying a large code block.

My suggestions is to add it to Hyland Community, alongside other downloadable artefacts for 23.1. Less effort for the dev teams and _hopefully_ simpler.

Comment (anxumalo) [reply content-services/latest/admin/cluster.md:352]: I don't have access to upload to Hyland Community, so let's take the discussion in-house to finalize the details. I'll ping you in a few minutes :-)

Comment (dsibilio) [reply content-services/latest/admin/cluster.md:352]: Moved the script to community-repo along with the other required files. Adapted the ent-repo CI/CD as well.

File: content-services/latest/admin/cluster.md [hunk 2]
```diff
@@ -332,9 +333,58 @@ This topic describes the instructions for installing and configuring Solr nodes
...
+
+### Set up repository clustering via external Hazelcast
+
+Managing your own Hazelcast cluster allows you to scale repository and Hazelcast instances independently, while also allowing you to manage resources allocation on an individual level. This kind of setup provides more fine-grained control at the cost of more necessary preliminary work.
+
+1. Prepare a valid Hazelcast XML configuration file including all caches definitions required by the repository. This can be mostly auto-generated for you by doing the following:
+
+    1. Locate the `caches.properties` file within the `alfresco.war:WEB-INF\lib\alfresco-repository-*.jar` archive.
+    2. Copy the `caches.properties` file to an easily accessible directory.
+    3. Clone the [alfresco-enterprise-repo.git](https://github.com/Alfresco/alfresco-enterprise-repo.git){:target="_blank"} repository locally.
```
Comment (dsibilio) [inline content-services/latest/admin/cluster.md:351]:
```suggestion
    3. Clone the [alfresco-community-repo.git](https://github.com/Alfresco/alfresco-community-repo.git){:target="_blank"} repository locally.
```

**Review by dsibilio — CHANGES_REQUESTED** [review]

File: content-services/latest/admin/cluster.md (hunk 2 above)
Comment (dsibilio) [reply content-services/latest/admin/cluster.md:351]: The python script has been moved to community-repo: https://github.com/Alfresco/alfresco-community-repo/blob/master/repository/scripts/hazelcast-init/generate-hazelcast-config.py.

**Review by HeinRagas — APPROVED** [review]
Happy to have this new feature explained!

**Pull request merged**